apps/outlook-rooms-addin/src/app/rooms/find-space/find-space.component.ts

```diff
@@ -126,27 +128,39 @@ export class FindSpaceComponent extends BaseClass implements OnInit {
         await this._org.initialised.pipe(first((_) => !!_)).toPromise();
         await this._spaces.initialised.pipe(first((_) => !!_)).toPromise();
         await this._state.available_spaces.pipe(take(1)).toPromise();
+        this.spaces$ = this._state.available_spaces;
+
+        this.spaces$.subscribe((i) => console.log(i, 'avail spaces'));
```

Comment:
Might wanna remove debug tracing if no longer needed.
Future reference: it would be better to debug a observable with `.pipe( tap (i) => console.log(i, 'avail spaces'))` instead of creating another subscription in the page.